draft-ietf-rats-architecture.md

```diff
@@ -898,10 +899,10 @@ some way of verifying the signature on the Evidence.  This may be in the form of
 The nature of how the Verifier manages to validate the signatures produced by the Attester is critical to the secure operation of an Attestation system, but is not the subject of standardization within this architecture.
 
 A conveyance protocol that provides authentication and integrity protection can be used
-to convey unprotected Evidence, assuming the following properties exists:
+to convey Evidence that is otherwise unprotected (e.g., not signed). Appropriate conveyance of unprotected Evidence (e.g., {{-uccs}}) relies on the following conveyance protocol's protection capabilities:
```

Comment:
```suggestion
to convey Evidence that is otherwise unsigned. Appropriate conveyance of unsigned Evidence (e.g., {{-uccs}}) relies on the following conveyance protocol's protection capabilities:
```
Ok as is but would prefer suggestion above if the meaning is the same, for brevity